src/Components/Question/Question.tsx

```diff
@@ -94,9 +88,7 @@ export const Question = (props: QuestionProps) => {
         Skip
       </button>
       <button
-        className='next-button'
-        disabled={!answer ? true: false}
-        type='button'
+        type="button"
```

Comment:
It looks like some of the classNames and the disable button code got removed from this file. @taylorjohnson141 Was this by mistake?